content/gophercon-2017/writing-a-successful-gophercon-proposal.md

```diff
@@ -0,0 +1,100 @@
++++
```

Comment:
It's the weirdo hugo front matter markup. When the page is rendered anything before the ++++ is ignored.